### PR TITLE
Add net.sourceforge.mcomix

### PR DESCRIPTION
**Pull request by catsout**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://sourceforge.net/p/mcomix/feature-requests/119/
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by catsout** [comment]
bot, build net.sourceforge.mcomix

**Comment by hfiguiere** [comment]
can't find the icon

File: net.sourceforge.mcomix.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/project/mcomix/MComix-2.0.2/mcomix-2.0.2.tar.gz
+        sha256: ef38d0713e56a07c72f98cc20c9eacda780e39f3b82fd7b1baa06a69c79c0a58
+
+      - type: patch
+        path: appdata.patch
+
+    modules:
+      - name: pygobject
```
Comment (hfiguiere) [inline net.sourceforge.mcomix.yml:89]: why not use the GNOME runtime instead? It has all that is needed, doesn't it?

**Comment by catsout** [comment]
bot, build net.sourceforge.mcomix

File: net.sourceforge.mcomix.yml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+rename-icon: mcomix
+rename-desktop-file: mcomix.desktop
+rename-appdata-file: mcomix.appdata.xml
+command: mcomix
+finish-args:
+  - --share=ipc
+  - --socket=x11
```
Comment (hfiguiere) [inline net.sourceforge.mcomix.yml:11]:
```suggestion
  - --fallback-socket=x11
```

File: net.sourceforge.mcomix.yml [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  # need, for own file chooser
+  - --filesystem=host:ro
```
Comment (hfiguiere) [inline net.sourceforge.mcomix.yml:16]: this is very broad.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: net.sourceforge.mcomix.yml (hunk 3 above)
Comment (catsout) [reply net.sourceforge.mcomix.yml:16]: use `home:ro`?

File: net.sourceforge.mcomix.yml (hunk 1 above)
Comment (catsout) [reply net.sourceforge.mcomix.yml:94]: I think freedesktop is enough, only needs py binding dependencies which are pretty small.

**Comment by catsout** [comment]
bot, build net.sourceforge.mcomix

**Pull request closed without merging**